cli/cmd/patchies/main.go

```
@@ -0,0 +1,90 @@
+		value, err := prompt(reader, output, "Mount path")
+		if err != nil {
+			return "", "", err
+		}
+		*path = value
+	}
+
+	return *token, *path, nil
+}
+
+func prompt(reader *bufio.Reader, output io.Writer, label string) (string, error) {
+	fmt.Fprintf(output, "%s: ", label)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)